WordPress/Sniffs/Arrays/ArrayAssignmentRestrictionsSniff.php

```diff
@@ -84,56 +82,57 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr )
 			return;
 		}
 
-		$tokens = $phpcsFile->getTokens();
-		$token = $tokens[ $stackPtr ];
+		$tokens  = $phpcsFile->getTokens();
+		$token   = $tokens[ $stackPtr ];
 		$exclude = explode( ',', $this->exclude );
 
-		if ( in_array( $token['code'], array( T_CLOSE_SQUARE_BRACKET ) ) ) {
-			$equal = $phpcsFile->findNext( T_WHITESPACE, $stackPtr + 1, null, true );
-			if ( $tokens[$equal]['code'] !== T_EQUAL ) {
+		if ( in_array( $token['code'], array( T_CLOSE_SQUARE_BRACKET ), true ) ) {
+			$equal = $phpcsFile->findNext( T_WHITESPACE, ( $stackPtr + 1 ), null, true );
+			if ( T_EQUAL !== $tokens[ $equal ]['code'] ) {
 				return; // This is not an assignment!
 			}
 		}
 
-		// Instances: Multi-dimensional array, keyed by line
+		// Instances: Multi-dimensional array, keyed by line.
 		$inst = array();
 
-		// $foo = array( 'bar' => 'taz' );
-		// $foo['bar'] = $taz;
-		if ( in_array( $token['code'], array( T_CLOSE_SQUARE_BRACKET, T_DOUBLE_ARROW ) ) ) {
-			if ( $token['code'] == T_CLOSE_SQUARE_BRACKET ) {
-				$operator = $phpcsFile->findNext( array( T_EQUAL ), $stackPtr + 1 );
-			} elseif ( $token['code'] == T_DOUBLE_ARROW ) {
+		/*
+		   Covers:
+		   $foo = array( 'bar' => 'taz' );
+		   $foo['bar'] = $taz;
+		 */
```

Comment:
> Just wondering if there is any rhyme or reason to it.

I ran the codebase against WP-Docs as well and fixed anything which was a 'quick' fix to make.

In this case - if I remember correctly -, it flagged this as a multi-line comment not using multi-line comment syntax.